### ISSUE TITLE
assistant is no longer triggered

### ISSUE THREAD

**Issue by mpatca** [RealRaven2000/quickFilters#173]
Hi,

the assistant is no longer triggered, for example when copying a file to a folder.  In the QF settings, activating the assistant on program startup is checked.  (Windows 10, Betterbird 102.9.1-bb32 64-Bit)

**Comment by RealRaven2000** [comment]
Does the status change, like this:

![image](https://user-images.githubusercontent.com/6314674/229381090-480cc57c-495c-4828-9111-338fbfafadfd.png)

it works for me - what folder are you  moving to? Please check the error log [ Ctrl+Shift+J ]

![image](https://user-images.githubusercontent.com/6314674/229381144-69c17399-a007-4acc-b19a-5a633b2c5535.png)

You can also activate debug mode on quickFilters Settings / advanced bottom right :

![image](https://user-images.githubusercontent.com/6314674/229381185-e86bb524-7871-4ccd-ba75-c0c3d91b8af6.png)

**Comment by mpatca** [comment]
Thanks for the quick reply. 

Until the last update, QF also worked for me.

However, I have found a solution. Before I posted the bug here, I had uninstalled QF and reinstalled it directly. That did not change anything. 
Just now I restarted Betterbird after the QF uninstallation and only then reinstalled QF - and look: It works perfectly again, plus the icon looks a bit different, as mentioned in the changelog. 

So maybe an update error on my side (especially since my computer would have to cope with a few crashes lately)?
